.github/workflows/fedora-rawhide-test.yml

```diff
@@ -9,4 +9,6 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Run Fedora Rawhide Test
-      run: sudo -E make -C scripts/ci fedora-rawhide CONTAINER_RUNTIME=podman BUILD_OPTIONS="--security-opt seccomp=unconfined"
+      # Make sure that XDG_RUNTIME_DIR environment variable is not set.
+      # https://github.com/checkpoint-restore/criu/issues/1942
```

Comment:
pls don't add issue in the code if it isn't todo/fixme. The comment has to contain all required info to understand the code.

Reply:
Thanks Andrei, I have updated the comment with FIXME for the GitHub issue in Podman.